packages/code/src/Code/Code.styles.ts

```diff
@@ -98,6 +124,22 @@ export const singleLineCodeWrapperStyle = css`
   padding-bottom: ${(singleLineComponentHeight - lineHeight) / 2}px;
 `;
 
+export function getExpandableCodeWrapperStyle(
+  expanded: boolean,
+  baseFontSize: number,
+) {
+  const codeLineHeight = baseFontSize === BaseFontSize.Body2 ? 24 : 20;
+  const numOfLinesWhenCollapsed = 5;
```

Comment:
Will this always be 5?

Reply:
Yup. If there are less than 5 lines of code there won't be an expand/collapse button, even if `expandable` is `true`. So the collapsed version will always be 5 lines when visible.

Reply:
Can you add this to the tsdoc and README?

Reply:
Also, remember to add a changeset